gulpfile.coffee

```diff
@@ -499,19 +499,37 @@ gulp.task "images", ->
 #    .pipe(header(banner, file: {path: 'foo'}))
 #    .pipe(gulp.dest(DIST_PATH))
 
+gulp.task "package-no-min:dist", ->
```

Comment:
I'm certain there is a better way to do this without duplicating almost all the functionality of `package:dist` but this was a good up front solution to get both things building and outputting.